src/test/testType.ts

```diff
@@ -108,6 +110,12 @@ export class TestTypeImpl {
   }
 
   private _modifier(type: 'skip' | 'fail' | 'fixme' | 'slow', location: Location, ...modifierArgs: [arg?: any | Function, description?: string]) {
+    // skip is allowed to create a test if the first parameter is the title.
```

Comment:
Let's move this to the `if (suite)` block - this way we won't get unexpected message like "test() can only be called in a test file"